### PR TITLE
[Add-ins](Testing) IE11 command line tooling

### PR DESCRIPTION
**Pull request by mscharlock**
This is in draft for the moment as I have to add it to a couple places as a link and in the table of contents, but please go ahead and review the text :) Thanks!

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[9db0df8](https://github.com/OfficeDev/office-js-docs-pr/commits/9db0df82fec36eec2b8bdee4a8d55fecb984d2fd)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/testing/ie-11-testing.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-ie11/docs/testing/ie-11-testing.md) | :warning:Warning | | [Details](#user-content-8ab9d7ca-475f98a2)

<a id="8ab9d7ca-475f98a2"></a>
### [docs/testing/ie-11-testing.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-ie11/docs/testing/ie-11-testing.md)
 - **Line 7**: **[Warning]** `````Markdown content is not allowed before H1.`````
 - **Line 8**: **[Suggestion]** `````H1 is required. Use a single hash (#) followed by a space to create your top-level heading. NOTE: This Suggestion will become a Warning on 05/15/2020.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C5%5C14%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202005141857211230-1851%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=Phi2eMR9n92i%2BajV3JXzUCr1iDsDfSLbFhRh7Gff3SA%3D&st=2020-05-14T18%3A54%3A06Z&se=2020-06-14T18%3A59%3A06Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[66368c9](https://github.com/OfficeDev/office-js-docs-pr/commits/66368c9001ced07dd64df5f867bc62f866c77e87)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/testing/ie-11-testing.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-ie11/docs/testing/ie-11-testing.md) | :warning:Warning | | [Details](#user-content-8ab9d7ca-0242973b)

<a id="8ab9d7ca-0242973b"></a>
### [docs/testing/ie-11-testing.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-ie11/docs/testing/ie-11-testing.md)
 - **Line 7**: **[Warning]** `````Markdown content is not allowed before H1.`````
 - **Line 8**: **[Suggestion]** `````H1 is required. Use a single hash (#) followed by a space to create your top-level heading. NOTE: This Suggestion will become a Warning on 05/15/2020.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C5%5C14%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202005142232070334-1851%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=ghEJMT5vt0ySuxdE%2F8903HmS%2BLu9dhzdzkD04rG15FQ%3D&st=2020-05-14T22%3A28%3A52Z&se=2020-06-14T22%3A33%3A52Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/testing/ie-11-testing.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+- [Node.js](https://nodejs.org/) (the latest [LTS](https://nodejs.org/about/releases) version)
+- A code editor. We recommend [Visual Studio Code](https://code.visualstudio.com/)
+- [Be part of the Office Insider program](https://insider.office.com)
+
+These instructions assume you have set up a Yo Office generator project before. If you haven't done this before, consider reading a quick start, such as [this one for Excel add-ins](../quickstarts/excel-quickstart-jquery.md).
+
+## Using IE11 tooling
+
+1. Create a Yo Office generator project. It doesn't matter what kind of project you select, this tooling will work with all project types.
```
Comment (nokafor) [inline docs/testing/ie-11-testing.md:25]: I think we should we add a note saying that people can install the package directly from npm if they already have an existing project.

File: docs/testing/ie-11-testing.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+---
+ms.date: 05/14/2020
+description: 'Test your add-in using Internet Explorer 11.'
```
Comment (lindalu-MSFT) [inline docs/testing/ie-11-testing.md:3]:
```suggestion
description: 'Test your Office Add-in using Internet Explorer 11.'
```

File: docs/testing/ie-11-testing.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+title: Internet Explorer 11 testing
```
Comment (lindalu-MSFT) [inline docs/testing/ie-11-testing.md:4]:
```suggestion
title: Test your Office Add-in using Internet Explorer 11
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[2e3b512](https://github.com/OfficeDev/office-js-docs-pr/commits/2e3b512b3f0c473fb5b0e406262383f52c0f575b)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/testing/ie-11-testing.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-ie11/docs/testing/ie-11-testing.md) | :warning:Warning | | [Details](#user-content-8ab9d7ca-ebb2536b)

<a id="8ab9d7ca-ebb2536b"></a>
### [docs/testing/ie-11-testing.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-ie11/docs/testing/ie-11-testing.md)
 - **Line 7**: **[Warning]** `````Markdown content is not allowed before H1.`````
 - **Line 8**: **[Warning]** `````H1 is required. Use a single hash (#) followed by a space to create your top-level heading.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C5%5C16%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202005161657203971-1851%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=TIuPch7y7QNEwQIlwvS4JsunUMIJe9DITXThK6SYbAE%3D&st=2020-05-16T16%3A53%3A41Z&se=2020-06-16T16%3A58%3A41Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[218e913](https://github.com/OfficeDev/office-js-docs-pr/commits/218e9137325c977b172b7d2ab68f1b99f05cb02a)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-deccbff2)
[_dependentPackages/CommonPlugins/tools/JoinTOC.ps1](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-ie11/_dependentPackages/CommonPlugins/tools/JoinTOC.ps1) | :x:Error | | [Details](#user-content-402d9d56-deccbff2)

<a id="deccbff2"></a>
###
 - **[Error]** 
`````
YamlDotNet.Core.YamlException: (Line: 1, Col: 1, Idx: 0) - (Line: 1, Col: 1, Idx: 0): No node deserializer was able to deserialize the node into type Microsoft.DocAsCode.DataContracts.Common.TocItemViewModel, Microsoft.DocAsCode.DataContracts.Common, Version=2.37.2.0, Culture=neutral, PublicKeyToken=null
   at YamlDotNet.Serialization.ValueDeserializers.NodeValueDeserializer.DeserializeValue(IParser parser, Type expectedType, SerializerState state, IValueDeserializer nestedObjectDeserializer)
   at Microsoft.DocAsCode.YamlSerialization.YamlDeserializer.LooseAliasValueDeserializer.DeserializeValue(IParser reader, Type expectedType, SerializerState state, IValueDeserializer nestedObjectDeserializer)
   at Microsoft.DocAsCode.YamlSerialization.YamlDeserializer.Deserialize(IParser parser, Type type, IValueDeserializer deserializer)
   at Microsoft.DocAsCode.YamlSerialization.YamlDeserializer.Deserialize[T](TextReader input, IValueDeserializer deserializer)
   at Microsoft.DocAsCode.Common.YamlUtility.Deserialize[T](String path)
   at Microsoft.OpenPublishing.PluginHelper.TocHelper.TryParseTocAsObjectModel(String tocPath, TocItemViewModel& toc)
   at Microsoft.OpenPublishing.PluginHelper.TocHelper.InjectTocMetadata(String tocPath, String metaName, String metaValue)
   at JoinTOC.TOCMerger.Merge(CommandLineOptions opt)
   at JoinTOC.Program.Main(String[] args)
`````

<a id="402d9d56-deccbff2"></a>
### [_dependentPackages/CommonPlugins/tools/JoinTOC.ps1](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-ie11/_dependentPackages/CommonPlugins/tools/JoinTOC.ps1)
 - **Line 108**: **[Error]** `````Error happened when executing plug-in script _dependentPackages/CommonPlugins/tools/JoinTOC.ps1 at step docset_prebuild. Please check your .openpublishing.publish.config.json?on the configuration of the plug-ins or ask for help?in Docs Support teams channel. Detailed error message: {"message":"YamlDotNet.Core.YamlException: (Line: 1, Col: 1, Idx: 0) - (Line: 1, Col: 1, Idx: 0): No node deserializer was able to deserialize the node into type Microsoft.DocAsCode.DataContracts.Common.TocItemViewModel, Microsoft.DocAsCode.DataContracts.Common, Version=2.37.2.0, Culture=neutral, PublicKeyToken=null\r\n   at YamlDotNet.Serialization.ValueDeserializers.NodeValueDeserializer.DeserializeValue(IParser parser, Type expectedType, SerializerState state, IValueDeserializer nestedObjectDeserializer)\r\n   at Microsoft.DocAsCode.YamlSerialization.YamlDeserializer.LooseAliasValueDeserializer.DeserializeValue(IParser reader, Type expectedType, SerializerState state, IValueDeserializer nestedObjectDeserializer)\r\n   at Microsoft.DocAsCode.YamlSerialization.YamlDeserializer.Deserialize(IParser parser, Type type, IValueDeserializer deserializer)\r\n   at Microsoft.DocAsCode.YamlSerialization.YamlDeserializer.Deserialize[T](TextReader input, IValueDeserializer deserializer)\r\n   at Microsoft.DocAsCode.Common.YamlUtility.Deserialize[T](String path)\r\n   at Microsoft.OpenPublishing.PluginHelper.TocHelper.TryParseTocAsObjectModel(String tocPath, TocItemViewModel& toc)\r\n   at Microsoft.OpenPublishing.PluginHelper.TocHelper.InjectTocMetadata(String tocPath, String metaName, String metaValue)\r\n   at JoinTOC.TOCMerger.Merge(CommandLineOptions opt)\r\n   at JoinTOC.Program.Main(String[] args)","source":"","file":null,"line":null,"message_severity":0,"date_time":"2020-05-16T17:04:16.7453736Z","code":"OPSPlugins_InternalError","log_item_type":1}.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C5%5C16%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202005161703329152-1851%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=600pyYUS7GTvAZD2TV1rmJEvymOjV8omxitgnuLAyOA%3D&st=2020-05-16T16%3A59%3A21Z&se=2020-06-16T17%3A04%3A21Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[c862fed](https://github.com/OfficeDev/office-js-docs-pr/commits/c862fedcaa129a6a46621fcb9477964ab77beb7c)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-7373d7b4)
[_dependentPackages/CommonPlugins/tools/JoinTOC.ps1](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-ie11/_dependentPackages/CommonPlugins/tools/JoinTOC.ps1) | :x:Error | | [Details](#user-content-402d9d56-7373d7b4)

<a id="7373d7b4"></a>
###
 - **[Error]** 
`````
YamlDotNet.Core.YamlException: (Line: 1, Col: 1, Idx: 0) - (Line: 1, Col: 1, Idx: 0): No node deserializer was able to deserialize the node into type Microsoft.DocAsCode.DataContracts.Common.TocItemViewModel, Microsoft.DocAsCode.DataContracts.Common, Version=2.37.2.0, Culture=neutral, PublicKeyToken=null
   at YamlDotNet.Serialization.ValueDeserializers.NodeValueDeserializer.DeserializeValue(IParser parser, Type expectedType, SerializerState state, IValueDeserializer nestedObjectDeserializer)
   at Microsoft.DocAsCode.YamlSerialization.YamlDeserializer.LooseAliasValueDeserializer.DeserializeValue(IParser reader, Type expectedType, SerializerState state, IValueDeserializer nestedObjectDeserializer)
   at Microsoft.DocAsCode.YamlSerialization.YamlDeserializer.Deserialize(IParser parser, Type type, IValueDeserializer deserializer)
   at Microsoft.DocAsCode.YamlSerialization.YamlDeserializer.Deserialize[T](TextReader input, IValueDeserializer deserializer)
   at Microsoft.DocAsCode.Common.YamlUtility.Deserialize[T](String path)
   at Microsoft.OpenPublishing.PluginHelper.TocHelper.TryParseTocAsObjectModel(String tocPath, TocItemViewModel& toc)
   at Microsoft.OpenPublishing.PluginHelper.TocHelper.InjectTocMetadata(String tocPath, String metaName, String metaValue)
   at JoinTOC.TOCMerger.Merge(CommandLineOptions opt)
   at JoinTOC.Program.Main(String[] args)
`````

<a id="402d9d56-7373d7b4"></a>
### [_dependentPackages/CommonPlugins/tools/JoinTOC.ps1](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-ie11/_dependentPackages/CommonPlugins/tools/JoinTOC.ps1)
 - **Line 108**: **[Error]** `````Error happened when executing plug-in script _dependentPackages/CommonPlugins/tools/JoinTOC.ps1 at step docset_prebuild. Please check your .openpublishing.publish.config.json?on the configuration of the plug-ins or ask for help?in Docs Support teams channel. Detailed error message: {"message":"YamlDotNet.Core.YamlException: (Line: 1, Col: 1, Idx: 0) - (Line: 1, Col: 1, Idx: 0): No node deserializer was able to deserialize the node into type Microsoft.DocAsCode.DataContracts.Common.TocItemViewModel, Microsoft.DocAsCode.DataContracts.Common, Version=2.37.2.0, Culture=neutral, PublicKeyToken=null\r\n   at YamlDotNet.Serialization.ValueDeserializers.NodeValueDeserializer.DeserializeValue(IParser parser, Type expectedType, SerializerState state, IValueDeserializer nestedObjectDeserializer)\r\n   at Microsoft.DocAsCode.YamlSerialization.YamlDeserializer.LooseAliasValueDeserializer.DeserializeValue(IParser reader, Type expectedType, SerializerState state, IValueDeserializer nestedObjectDeserializer)\r\n   at Microsoft.DocAsCode.YamlSerialization.YamlDeserializer.Deserialize(IParser parser, Type type, IValueDeserializer deserializer)\r\n   at Microsoft.DocAsCode.YamlSerialization.YamlDeserializer.Deserialize[T](TextReader input, IValueDeserializer deserializer)\r\n   at Microsoft.DocAsCode.Common.YamlUtility.Deserialize[T](String path)\r\n   at Microsoft.OpenPublishing.PluginHelper.TocHelper.TryParseTocAsObjectModel(String tocPath, TocItemViewModel& toc)\r\n   at Microsoft.OpenPublishing.PluginHelper.TocHelper.InjectTocMetadata(String tocPath, String metaName, String metaValue)\r\n   at JoinTOC.TOCMerger.Merge(CommandLineOptions opt)\r\n   at JoinTOC.Program.Main(String[] args)","source":"","file":null,"line":null,"message_severity":0,"date_time":"2020-05-16T17:08:54.8273504Z","code":"OPSPlugins_InternalError","log_item_type":1}.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C5%5C16%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202005161708090372-1851%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=dh6zq1K7%2Fk2yDyy7FbwNaN%2B8mZJhdg51q5aprHaiXDE%3D&st=2020-05-16T17%3A04%3A00Z&se=2020-06-16T17%3A09%3A00Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[1c63ad2](https://github.com/OfficeDev/office-js-docs-pr/commits/1c63ad269fd6caf3e234e5518cb641f972ddf221)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/testing/ie-11-testing.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-ie11/docs/testing/ie-11-testing.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-ie11/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C5%5C17%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202005172316404677-1851%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=jkHMo%2FQ9uJjaIfEcwrMpCzDdtHGeLb4cvlJsZ0j3bA4%3D&st=2020-05-17T23%3A13%3A23Z&se=2020-06-17T23%3A18%3A23Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by mscharlock** [comment]
Note to self: Ready for publish

**Pull request merged**